ql/src/java/org/apache/hadoop/hive/ql/ddl/table/constraint/add/AlterTableAddConstraintAnalyzer.java

```diff
@@ -77,9 +82,23 @@ protected void analyzeCommand(TableName tableName, Map<String, String> partition
       throw new SemanticException(ErrorMsg.NOT_RECOGNIZED_CONSTRAINT.getMsg(constraintNode.getToken().getText()));
     }
 
-    Constraints constraints = new Constraints(primaryKeys, foreignKeys, null, uniqueConstraints, null,
```

Comment:
According to the Hive checkstyle, the max line length is 120, please format the code according to that.

Reply:
Don't see any line more than 90 in this class.

Reply:
True, but I don't see the reason for this very change here, where you broke a line that was not longer than 120